### ISSUE TITLE
Dashboard empty (no pod, no service, no node) No resources found.

### ISSUE THREAD

**Issue by nicoclau** [kubernetes/dashboard#4842]
Hello I use kubernetes from docker for windows and the last dashboard 2.0.0 rc3
The dashboard doesnt display any ressource.

When I used a older dashboard it works.
I noticed the old dashboard uses a different namespace : kube-system
And the last dashboard uses the namespace : kubernetes-dashboard.

here the version of K8s 

Client Version: version.Info{Major:"1", Minor:"14", GitVersion:"v1.14.8", GitCommit:"211047e9a1922595eaa3a1127ed365e9299a6c23", GitTreeState:"clean", BuildDate:"2019-10-15T12:11:03Z", GoVersion:"go1.12.10", Compiler:"gc", Platform:"windows/amd64"}
Server Version: version.Info{Major:"1", Minor:"14", GitVersion:"v1.14.8", GitCommit:"211047e9a1922595eaa3a1127ed365e9299a6c23", GitTreeState:"clean", BuildDate:"2019-10-15T12:02:12Z", GoVersion:"go1.12.10", Compiler:"gc", Platform:"linux/amd64"}

I see the error :

namespaces is forbidden: User "system:anonymous" cannot list resource "namespaces" in API group "" at the cluster scope

Can you help me ?
Thanks

**Comment by floreks** [comment]
Yes, v2 is completely different and was moved to its' own namespace. Refer to the readme and completely remove old Dashboard first, before installing a new version. This is a configuration issue on your side.

**Comment by nicoclau** [comment]
Hello floreks thanks for your kind help.
I was thinking about the notion of namespace.

before it was kube-system.
I used to get the token from there to login inside the dashboard

kubectl -n kube-system describe secret default

Then I thought it is wrong, I must pick up the token from the proper namespace
So I used

kubectl -n **kubernetes-dashboard** describe secret default

It worked then 👍 

Is there a tutorial about dashboard login
There are two options:
- kubeconfig
- token

It is unclear how to use them.

Thanks for any link for logging.

Nicolas
